test/battleship/game/game_test.go

```diff
@@ -0,0 +1,9 @@
+package game
```

Comment:
This file does not test main.go

Reply:
I just bootstrap the run of an empty test. Real tests will be added in next commits.

Reply:
Ok so remove it or test something existing. Do no add useless files